src/Tribe/Service_Providers/Context.php

```diff
@@ -249,6 +249,15 @@ static function () {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return count( array_filter( [
+									! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ),
+								] ) );
```

Comment:
You either return a value, or return `Tribe__Context::NOT_FOUND` to indicate a value was not found.
`false` does not mean "not found" in the Context, it means "false".